cfme/tests/storage/test_volume.py

```diff
@@ -224,7 +227,30 @@ def test_storage_volume_attached_delete():
             4. check for flash message " Cloud Volume "Volume_name" cannot be
             removed because it is attached to one or more Instances "
         """
-    pass
+    volume = create_volume(appliance, provider, from_manager, az=instance_fixture.
```

Comment:
As far as I can tell, this is not part of the test -- thoughts on moving it to a fixture

Reply:
Well I need to know availiability zone of instance to create volume. Can I do it as secondary fixture somehow?

Reply:
@mmojzis you can pass instance_fixture to the fixture that will create the volume.

```python,
@pytest.fixture
def volume(appliance, provider, from_manager, instance_fixture):
    yield create_volume(appliance, provider, from_manager, az=instance_fixture.                       
                vm_default_args["environment"]["availability_zone"], should_assert=True)
    # do cleanup
```

Reply:
Okay, reworked.